TribalWars/src/ApplicationCore/Results/ServiceResult.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+
+namespace ApplicationCore.Results
+{
+    public class ServiceResult
+    {
+        public IEnumerable<string> Errors { get; }
+        public bool Succeeded { get; }
+
+        public ServiceResult(bool succeeded, IEnumerable<string> errors)
```

Comment:
One interesting idea in this situation is to define enum with different statuses, like:
```
public enum ServiceStatus
    {
        Undefined = 0,
        Success = 1,
        Failure = 2,
        Exception = 3
    }
}
```

You can then handle your ServiceResult with a single constructor instead of using multiple static classes.
Plus, you can add a helper for Success status if you feel like it.

```
public class ServiceResult
{
public IEnumerable<string> Errors { get; }
public ServiceStatus Status { get; }

public ServiceResult(ServiceStatus status, IEnumerable<string> errors)
{
Status = status;
Errors = errors;
}
public bool IsSuccessful => Status == ServiceStatus.Success;
}
```

This way it's easier to implement more statuses in the future.

Reply:
It's a good idea but I'd rather leave those static methods that return new object of ServiceResult with predefined status because we can just use e.q.:
`ServiceResult.Success(); `
instead of: 
`new ServiceResult(ServiceResultStatus.Success, null);`